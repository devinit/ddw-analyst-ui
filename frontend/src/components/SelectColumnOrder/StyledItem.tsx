import styled from 'styled-components';

export const StyledItem = styled.div`
  margin-top: 4px;
  margin-bottom: 4px;
  margin-right: 8px;
  cursor: grab;
  word-wrap: break-word;
  line-height: 1em;
  white-space: normal;
  padding: 0.78571429em 2.1em 0.78571429em 1em;
  color: rgba(0, 0, 0, 0.87);
  border: 1px solid rgba(34, 36, 38, 0.15);
  border-radius: 0.28571429rem;
  min-height: 2.71428571em;
  max-width: 500px;
  min-width: 300px;
  width: fit-content;
`;

export default StyledItem;
