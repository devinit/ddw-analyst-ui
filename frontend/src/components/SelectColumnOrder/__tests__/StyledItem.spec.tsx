import 'jest-styled-components';
import React from 'react';
import * as TestRenderer from 'react-test-renderer';
import { StyledItem } from '../StyledItem';

describe('StyledItem', () => {
  test('renders correctly with the required props', () => {
    const renderer = TestRenderer.create(<StyledItem />).toJSON();

    expect(renderer).toMatchSnapshot();
  });
});
