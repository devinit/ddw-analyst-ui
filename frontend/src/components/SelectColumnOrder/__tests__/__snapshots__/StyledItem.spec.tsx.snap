// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledItem renders correctly with the required props 1`] = `
<div
  className=""
/>
`;
